src/ripple/basics/impl/BasicConfig.cpp

```diff
@@ -134,6 +116,17 @@ BasicConfig::overwrite (std::string const& section, std::string const& key,
 }
 
 void
+BasicConfig::legacy(std::string const& section, std::string value)
+{
+    map_[section].legacy(std::move(value));
+}
+
+std::string BasicConfig::legacy(std::string const& sectionName) const
```

Comment:
Let's be consistent in this change at least whether a function signature is one line or two.  I personally prefer to have the return type on the same line unless it's super-long - is it really worth a line just to say `void`? - but be consistent either way.